3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -25,13 +25,21 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 
 function rollTheDices() {
   // TODO Refactor this function
-  const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  const dices = [
+    rollDice(1),
+    rollDice(2),
+    rollDice(3),
+    rollDice(4),
+    rollDice(5),
+  ];
+  return Promise.all(dices);
```

Comment:
You can write this a bit simpler like so
```suggestion
  const dices = [1, 2, 3, 4, 5];
  return Promise.all(dices.map(rollDice));
```